node/rpc/src/task.rs

```diff
@@ -59,3 +60,22 @@ where
 		})
 	}
 }
+
+#[cfg(test)]
+pub mod test {
+	use super::*;
+	use creditcoin_node_runtime::Block;
+
+	#[test]
+	fn offchain_nonce_key_works() {
+		let client = Arc::new(test_client::new());
+		let t = Task::<_, Block>::new(client, DenyUnsafe::No);
+		//$ ./node key inspect //Alice
+		t.offchain_nonce_key("5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY".into())
+			.unwrap();
+		t.offchain_nonce_key(
+			"0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d".into(),
+		)
+		.unwrap();
+	}
+}
```

Comment:
So is this just testing that `offchain_nonce_key` doesn't return an `Err`?

Reply:
+1 on the question. Ideally there would be some assertions as well.

Reply:
Ah, well I agree that it is incomplete. I was hoping to provide the basics (test client working and minimal test example) and either Alex taking over or requesting proper tests. Going back to the question. Yes, as it is, it only checks that given the expected input that it doesn't panic.

Reply:
Alright, let's clean the rest of the comments up and I can work on adding more assertions and probably more tests.